sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/QuantileSummariesSuite.scala

```diff
@@ -169,5 +170,22 @@ class QuantileSummariesSuite extends SparkFunSuite {
       checkQuantile(0.1, data, s)
       checkQuantile(0.001, data, s)
     }
+
+    // length of data21 is 4 * length of data22
+    val data21 = data.zipWithIndex.filter(_._2 % 5 != 0).map(_._1).toSeq
```

Comment:
Total nit, but you can: `val (data21, data22) = data.zipWithIndex.partition(_._2 % 5 != 0).map(_._1)`, I believe

Reply:
Hello @srowen , thanks a lot for the remarks. I've fixed the other two, but for this one, if I understood the usage correctly, I need to do something on the lines of:

```scala
val (iterData21, iterData22) = data.zipWithIndex.partition(_._2 % 5 != 0)
val data21 = iterData21.map(_._1).toSeq
val data22 = iterData22.map(_._1).toSeq
```

Or is there a cleaner way to do it?

Reply:
Nah, disregard this then if .toSeq is necessary